src/react/src/components/FacilityDetailsClaimedInfo.jsx

```diff
@@ -62,7 +62,7 @@ export default function FacilityDetailsClaimedInfo({ data, formatListItem }) {
                 label="Minimum Order"
             />
             <ClaimInfoSection
-                value={facility.average_lead}
+                value={facility.average_lead_time}
```

Comment:
I don't see such a name in serializers.

Reply:
@vladsha-dev check [here](https://github.com/opensupplyhub/open-supply-hub/blob/main/src/django/api/serializers/facility/approved_facility_claim_serializer.py#L35)

Reply:
Basically, error was on the FE part. Average lead time correctly saves to the database.

Reply:
Ok, thanks!